src/components/molecules/dropdownMenu/dropdownMenu.module.scss

```diff
@@ -27,3 +29,42 @@ menuContainer {
 .menu li {
   border-bottom: 1px solid #dddddd;
 }
+
+.menuWrapper {
+  max-width: 400px;
+  position: relative;
+  border-radius: 3px;
+  border: solid 2px $input-border;
+  width: 100%;
+  background-color: $input-background;
+}
+.activeElement {
+  padding: 5px 15px 5px 15px;
```

Comment:
I think we should use more responsive units

Reply:
Should be good now. Changed ~all units to responsive ones, cleanup unused css & tweaked conditional rendering more compact.